docs/source/format/CanonicalExtensions.rst

```diff
@@ -72,4 +72,26 @@ same rules as laid out above, and provide backwards compatibility guarantees.
 Official List
 =============
 
-No canonical extension types have been standardized yet.
+Fixed size tensor
+=================
+
+* Extension name: `arrow.fixed_size_tensor`.
+
+* The storage type of the extension: ``FixedSizeList``.
```

Comment:
Can you describe the value type and the size of the fixed-size list?

Reply:
@AlenkaF I think we should probably describe the size in context of the tensor (i.e. the product of the shape), instead of generic explanation of FixedSizeList type. Also "size of the child array (individual tensor)." isn't fully correct, since the "child array" is generally considered to be the full child array (which would have a size of "(length of TensorArray) x (size of each individual tensor)")

Reply:
Oh, thanks for explaining! Will correct 👍

Reply:
Changed the description in https://github.com/apache/arrow/pull/33925/commits/37e83db3787b4a94a359c577b04e4583f7f60818